checks/microbenchmarks/cpu/stream/stream.py

```diff
@@ -14,94 +14,64 @@ class StreamTest(rfm.RegressionTest):
        Triad:          13991.7     0.017174     0.017153     0.017192
     '''
 
-    def __init__(self):
-        self.descr = 'STREAM Benchmark'
-        self.exclusive_access = True
-        self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc',
-                              'arolla:cn', 'arolla:pn', 'tsa:cn', 'tsa:pn']
-        self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-gnu',
-                                    'PrgEnv-intel', 'PrgEnv-pgi',
-                                    'PrgEnv-nvidia']
+    descr = 'STREAM Benchmark'
```

Comment:
I would:
1/ update the Copyright 2016-2022 to 2016-2023
2/ add a reference to https://www.cs.virginia.edu/stream/FTP/Code/ (after line 14) to make it clear where the code is from